CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CheckDrive.Web.Stores.Debts;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CheckDrive.Web.Controllers
+{
+    public class DebtsController : Controller
```

Comment:
should be better to be semicolon at the end of namespace
![image](https://github.com/user-attachments/assets/1b1620a0-0d6e-4204-a6dc-faf6e97c2bf5)

Reply:
like this

Reply:
fixed